api/src/main/resources/liquibase.xml

```diff
@@ -71,4 +71,12 @@
 
     </changeSet>
 
-</databaseChangeLog>
\ No newline at end of file
+    <changeSet id="${project.parent.artifactId}-20230323-1212" author="fdeniger">
+        <createIndex tableName="${project.parent.artifactId}_entity_basis_map" indexName="${project.parent.artifactId}_entity_basis_index">
```

Comment:
For the changeset id, i understand how this will work when `${project.parent.artifactId}` is not substituted with the expected value. But for a table whose name starts with the literal `${project.parent.artifactId}`, i totally have no idea how this will magically work. 😊

Reply:
It *should* be being processed by Maven's [resource filtering](https://github.com/openmrs/openmrs-module-datafilter/blob/af6cca0e910a79d6a37e768ea30736fc93f89a43/api/pom.xml#L52) capabilities that we're apparently using. Tbh, I made that substitution (the original PR had `datafilter`), but I noticed that [we already use that variable when creating the table](https://github.com/openmrs/openmrs-module-datafilter/blob/af6cca0e910a79d6a37e768ea30736fc93f89a43/api/src/main/resources/liquibase.xml#L35), so the idea was just to be consistent.

When I build the module locally and checkout the output in the JAR it's this:

```
    <changeSet id="datafilter-20230323-1212" author="fdeniger">
        <createIndex tableName="datafilter_entity_basis_map" indexName="datafilter_entity_basis_index">
            <column name="entity_type"/>
            <column name="basis_type"/>
            <column name="basis_identifier"/>
        </createIndex>
    </changeSet>
```

Reply:
Makes sense. Thanks! :)